one_fm/templates/pages/job_application.html

```diff
@@ -9,7 +9,7 @@
 {% endblock %}
 
 {% block header %}
-  <h5 id="job_opening" data="{{ job_opening.name }}">Dear Applicant for the postion of {{ job_opening.designation }}, Enter your Full Name in English.</h5>
+  <h5 id="job_opening" data="{{ job_opening.name }}">Dear applicant for the postion of the {{ job_opening.designation }}, enter your full name in english.</h5>
```

Comment:
"Dear applicant, for the position {designation}, enter your full name in English."